app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -50,6 +51,11 @@ BUILD_PREREQUIRES="
 
 defineDebugInfoPackage wordgrinder$secondaryArchSuffix \
 	$commandBinDir/wordgrinder
```

Comment:
Add empty line here

Reply:
> Add empty line here

Oops, this was an old message which @Begasus had added when an empty line was missing. But the message was not attached to the correct line, so GitHub continued showing it.
But it was OK before this last change. One empty line is enough, after the "fi" and before the "defineDebugInfoPackage". (No need for 2 blank lines.)